### ISSUE TITLE
Advantages compared to similar tools & no mentions of rtags?

### ISSUE THREAD

**Issue by Silex** [ffevotte/clang-tags#12]
Hello,

In the "Similar tools" of the README, I was surprised not to see https://github.com/Andersbakken/rtags

It's also hard to know wether clang-tags has improvements over rtags... but probably I didn't search the documentation hard enough.

**Comment by ffevotte** [comment]
Yes, you're right. At the time I started working on `clang-tags` I didn't know of `rtags` (probably did not use the right keywords when I searched for clang-based indexing tools). I only recently learned about this project by looking at the starred projects by one of `clang-tags`'s followers (thank you github).

I honestly have not spent too much time trying `rtags`, but here is what I can
say from my limited experience:
- had I known about it sooner, I would never have started developing
  `clang-tags`.
- `rtags` is much more advanced and featureful than `clang-tags`; I don't know
  of any `clang-tags` feature not implemented in `rtags`, except maybe the
  `strace`-based system allowing to build compilation databases.
- The only thing which prevented me from using it more is the rather long list
  of dependencies, which is incompatible with the system I have to use at work.
